### PR TITLE
fix: Move react-i13n to dependencies since it's used in component-lib…

### PR DESCRIPTION
**Pull request by fabiosantoscode**
…rary

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -161,7 +162,6 @@
     "pre-commit": "^1.0.10",
     "react": "^0.14.6",
     "react-addons-test-utils": "^0.14.0",
-    "react-i13n": "^2.0.2",
```
Comment (umbobabo) [inline package.json]: i13n here is is optional and used just in the example.
This was the reason why I placed it on the dev dependencies.
Can you explain this change?

Comment (fabiosantoscode) [inline package.json]: example.es6 is used by component-library, causing a crash.

We either add react-i13n to devDependencies or add it to component-library.

Comment (umbobabo) [inline package.json]: This on the component-library is a side-effect. I think that should be fixed on the component-library more then here, but I don't have a strong opinion on that.
Maybe should be a peer-dependecy?

Comment (fabiosantoscode) [inline package.json]: Let's not have more peer dependencies ;)

I'm +1 for adding it to library instead. More components will be usable with react-i13n, and it's better to have a central hack than 5 hacks.

**Pull request closed without merging**